docs/advanced-config.md

```diff
@@ -55,3 +55,22 @@ The following config options can be set by passing them as arguments to `startTr
 - `instrumentations`: Can be used to enable additional instrumentation packages.
 
 - `captureHttpRequestUriParams`: Either a list of keys (case-sensitive) of HTTP query parameters to capture or a function that gets invoked with the current span and query parameters to set a custom span attribute. When using the former, parameters are set as span attributes as `http.request.param.${key}`. Attribute keys are normalized at capture time, meaning `.` is replaced with `_` to avoid any attribute namespacing issues.
+
+### Metrics
+
+| Environment variable<br>``startMetrics()`` argument             | Default value           | Support | Notes
+| --------------------------------------------------------------- | ----------------------- | ------- | ---
+| `SPLUNK_METRICS_ENABLED`<br>`enabled`                           | `false`                 | Experimental | Enabled metrics export. See [metrics documentation](metrics.md) for more information.
+| `SPLUNK_METRICS_ENDPOINT`<br>`endpoint`                         | `http://localhost:9943` | Experimental | The SignalFx metrics endpoint to send to.
+| `SPLUNK_METRICS_EXPORT_INTERVAL`<br>`exportInterval`            | `5000`                  | Experimental | The interval, in milliseconds, of metrics collection and exporting.
```

Comment:
Perhaps an extra column for the `startMetrics` argument would make it easier to scan the table.

Reply:
s/SignalFx/Splunk

Reply:
Added a link to SignalFx client. Followed the existing doc style and `startTracing` did not have the extra column either, should I split both of them to separate column?